build.gradle

```diff
@@ -53,7 +54,7 @@ configurations.all {
 // - define the validation
 // - define dependencies to include the generated code
 
-def openapiSourceFile = "${projectDir}/src/main/resources/service_openapi.yaml"
+def openapiSourceFile = "${projectDir}/src/main/resources/api/service_openapi.yaml"
```

Comment:
Why that change?

Reply:
I wanted to organize the resources/ directory a bit since it had both the API yaml and the UI html page. Either way works

Reply:
That makes sense. I was worried it was some weird "you have to put it here" artifact of the doc generation.